templates/oauth2/oauth2-login.html.twig

```diff
@@ -0,0 +1,144 @@
+
+{#
+ # Authorization Server Member
+ #
+ # @package   OpenEMR
+ # @link      http://www.open-emr.org
+ # @author    Jerry Padgett <sjpadgett@gmail.com>
+ # @copyright Copyright (c) 2020 Jerry Padgett <sjpadgett@gmail.com>
+ # @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
+{% extends "oauth2/oauth2-base.html.twig" %}
+{% block title %}{{ "OpenEMR Authorization"|xlt }}{% endblock %}
+{% block content %}
+<div class="row h-100 w-100 justify-content-center align-items-center">
+    <div class="col-sm-6 bg-light text-dark">
+        <div class="text-md-center">
+        {% if mfaRequired %}
+            <h4 class="mb-4 mt-1">{{'MFA Verification'|xlt }}</h4>
+        {% else %}
+            <h4 class="mb-4 mt-1">{{ "Sign In"|xlt  }}</h4>
+        {% endif %}
+        </div>
+        <hr />
+        <form method="post" name="userLogin" id="userLogin" action="{{ redirect|attr }}">
+        {% if invalid is not empty %}
+            <div class="alert alert-danger">
+                <p>{{ invalid|text }}</p>
+            </div>
+        {% endif %}
+            <input type="hidden" name="csrf_token_form" value="{{ csrfTokenRaw('oauth2')|attr }}" />
+            {% if not mfaRequired %}
+
+            {% if enforce_signin_email %}
+            <div class="form-group">
+                <input class="form-control" placeholder="{{ "Email if required"|xla  }}" type="email" name="email">
```

Comment:
How about including `label` tags to improve accessibility?

Reply:
Because I'm keeping the code as near as possible to the original UX where screenshots were taken by our certifying authority for our US ONC 2015 Certification.  If I add labels (which is my preference, but this was overlooked at the time this page was built) we put ourselves at risk of having to redo our FHIR API certification.

Reply:
Noted, thanks for the info!